virtool/http/policy.py

```diff
@@ -68,6 +68,12 @@ class PublicRoutePolicy(DefaultRoutePolicy):
     allow_unauthenticated = True
 
 
+class WebSocketRoutePolicy(DefaultRoutePolicy):
+    """Any client can access the route."""
+
+    allow_unauthenticated = True
```

Comment:
Is this not just the same as `PublicRoutePolicy`?

Reply:
Yes mostly, but with one annoying distinction. The auth functions checks for the `PublicRoutePolicy` and if it finds it, the function does not bother authenticating the client and just passes default values. The result of that is that I can't tell whether the user is logged in in websocket's `root` as the client object will always have default values. 

That said I could probably clean this up a little by inheriting from `publicRoutePolicy` and will do that with the next push if it works as expected

Reply:
Turns out if I make it a subclass then `isInstance` still considers `websocketRoutePolicy` an instance of `publicRoutePolicy` left mostly as is but with a more suitable docstring.